bakerydemo/static/css/main.css

```diff
@@ -235,341 +400,466 @@ cite {
-.logo:hover {
-  color: #d4566b;
-  text-decoration: none;
+
+.index-header__introduction.index-header__introduction--blog {
+  margin-bottom: 20px;
+}
+
+@media (min-width: 766px) {
+  .index-header__title {
+    font-size: 100px;
+    line-height: 64px;
+    margin-top: 70px;
+  }
+
+  .index-header__introduction {
+    font-size: 32px;
+    line-height: 42px;
+    margin-top: 40px;
+    margin-bottom: 40px;
+  }
+}
+
+
+@media (min-width: 766px) {
+  .index-header__title.index-header__title--blog {
+    line-height: 110px;
+  }
+}
+
+
+/* Main menu */
+.navigation {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+  align-items: center;
+  padding: 25px 0;
 }
-.search {
+
+
+.navigation__search {
   display: none;
   margin: 15px 0 0 0;
   position: relative;
 }
-@media (min-width: 768px) {
-  .search {
-    display: inline-block;
-    float: right;
-    margin: 0 0 0 30px;
-  }
-}
-.search-icon {
+
+.navigation__search-icon {
   display: block;
-  display: inline-block;
-  height: 20px;
   position: absolute;
-  right: 10px;
-  top: 10px;
-  width: 20px;
+  right: 13px;
+  top: 11px;
 }
-.search-icon svg {
-  fill: #d4566b;
-  height: 20px;
-  width: 20px;
+
+.navigation__search-input {
+  color: var(--dark);
+  font-size: 16px;
+  line-height: 24px;
+  border: 1px solid var(--dark);
+  padding: 10px;
 }
 
-/* Main menu */
-nav {
-  margin: 15px 0 5px;
+.navigation__items {
   display: none;
+  list-style: none;
+  margin: 0;
+  padding: 20px 0 0 0;
 }
-#main-navigation {
-  padding-left: 0;
-}
-@media (min-width: 768px) {
-  nav {
-    margin: 15px 0 0 0;
-    border-top: 1px solid rgba(255,255,255,0.1);
+
+@media (min-width: 1150px) {
+  .navigation {
+    flex-wrap: nowrap;
+  }
+
+  .navigation__search {
     display: block;
+    margin: 0 0 0 auto;
   }
-}
-.nav-pills>li+li {
-  margin-left: 0;
-}
-.nav-pills>li>a {
-  border-radius: 0;
-  border-top: 1px solid transparent;
-  color: white;
-  color: white;
-  font-family: 'Lato', sans-serif;
-  font-size: 11px;
-  font-weight: 300;
-  letter-spacing: 0.15em;
-  margin-top: -1px;
-  padding: 10px 10px;
-  text-transform: uppercase;
-}
-@media (min-width: 768px) {
-  .nav-pills>li>a {
-    padding: 10px 20px;
-    font-size: 14px;
+
+  .navigation__items {
+    padding: 0 0 0 20px;
   }
 }
-.nav-pills>li>a, .nav-pills>li>a:focus, .nav-pills>li>a:hover,
-.nav-pills>li, .nav-pills>li, .nav>li>a, .nav>li>a {
-  border-top: 1px solid transparent;
-}
-/* The following is to stop a pixel shift on hover */
-.nav-pills>.breads {
-  width: 90px;
-}
-.nav-pills>.locations {
-  width: 140px;
-}
-.nav-pills>.blog {
-  width: 86px;
-}
-.nav-pills>.gallery {
-  width: 115px;
+
+.navigation__items>li {
+  padding: 10px 0;
 }
-.nav-pills>.contactus {
-  width: 148px;
+
+.navigation__items>li>a {
+  color: var(--dark);
+  font-family: var(--font--secondary);
+  font-size: 18px;
+  line-height: 28px;
+  padding: 10px 0;
 }
-.nav-pills>.about {
-  width: 88px;
+
+@media (min-width: 1150px) {
```

Comment:
Could we define breakpoints with CSS variables so it’s clear which ones are consistent and which aren’t? and so they have defined names

Reply:
Unfortunately this isn't possible - I did originally try this earlier in development. Some more information about why: https://bholmes.dev/blog/alternative-to-css-variable-media-queries/